### PR TITLE
homework2

### PR DESCRIPTION
**Pull request by nazlukyanovich**
Homework

### CODE REVIEW

File: fraction.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    	result =(a - int(a)) * -10;
+    } else {
+    	result =(a - int(a)) * 10;
+    }
+    
+    cout << "The first digit of the fractional part of the number is " << result;  
+
+    return 0;
+}
+
```
Comment (AlexanderRusak) [inline fraction.cpp]: Incorrect output

File: guessTheNumber.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		if (b > a) {
+			cout << "Too much! Try less than it" << endl;
+		} else if (b < a) {
+			cout << "Too few! Try greater than it" << endl;
+		} else cout << "Bingo!" << endl;
+	}
+	while (b != a);
+	
+	return 0;
+}
```
Comment (AlexanderRusak) [inline guessTheNumber.cpp]: Prevent negative numbers from being entered

**Review by AlexanderRusak — CHANGES_REQUESTED** [review]

File: guessTheNumber.cpp (hunk 2 above)
Comment (AlexanderRusak) [reply guessTheNumber.cpp:25]: I've reviewed your tasks, but look more attentively on logic in your applications

**Comment by AlexanderRusak** [comment]
I've reviewed your tasks, but look more attentively on logic in your applications

**Comment by nazlukyanovich** [comment]
fixed, thanks

**Review by AlexanderRusak — APPROVED** [review]

**Pull request merged**